### PR TITLE
UG updates

### PR DESCRIPTION
**Pull request by alfaloo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -27,42 +27,92 @@ description and an overview of main
...
+#### macOS
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar MediCLI.jar` command to run the application.<br>
+1. Open finder and navigate to the home folder containing the MediCLI jar file.
+
+2. Double-click on the MediCLI application and it should start up!<br>
+
+    ![Ui](images/macOSStartup.png)
+
+#### CLI Alternative Solution
```
Comment (Kappaccinoh) [inline docs/UserGuide.md:80]: Alternative Solution through Terminal (Mac/Windows/Linux)

Comment (officialchengyud) [reply docs/UserGuide.md:80]: Agreed.

File: docs/images/GUI.png
Comment (officialchengyud) [inline docs/images/GUI.png:1]: I like the diagram. It is very clear.

**Review by officialchengyud — APPROVED** [review]
LGTM. Nice job Zhiyang.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -27,42 +27,92 @@ description and an overview of main
...
-can get started, encompassing installation
-instructions, compatibility with different
-operating systems, elements of the graphical
-user interface (GUI), and a tutorial on using
-the command-line interface (CLI).]
 
-### Compatibility
-[OS]
+Ready to step into the world of MediCLI? This section will provide detailed information on how users can get started,
+which includes basic system requirements, installation instructions, overview of the graphical interface (GUI),
```
Comment (ararchch) [inline docs/UserGuide.md]: Maybe we can change this from GUI to display? to make it more clear

**Review by ararchch — CHANGES_REQUESTED** [review]

**Review by ararchch — APPROVED** [review]
LGTM

**Pull request merged**